packages/link-lists/package.json

```diff
@@ -0,0 +1,73 @@
+{
+	"name": "@gov.au/link-lists",
+	"version": "0.1.0",
+	"description": "A simple list of inline links.",
+	"keywords": [
+		"uikit",
+		"uikit-module",
+		"scss",
+		"css",
+		"gov.au",
+		"Commonwealth of Australia",
+		"design guide",
+		"design system"
+	],
+	"scripts": {
+		"postinstall": "pancake batter \"$(cd .. && npm prefix)\" -v",
+
+		"prepublish": "node ../../helper.js compile publish",
+
+		"watch:sass": "node ../../helper.js compile && onchange 'src/**/*' 'tests/site/*.scss' -- node ../../helper.js compile",
+		"serve": "browser-sync start --server --startPath 'tests/site' --files 'tests/site/*.html, tests/site/*.css, tests/site/*.js'",
+		"watch": "npm-run-all --parallel serve watch:*"
+	},
+	"dependencies": {
+		"@gov.au/pancake": "latest",
+
+		"@gov.au/core": "0.1.0",
+		"@gov.au/body": "0.1.0"
+	},
+	"peerDependencies": {
+		"@gov.au/core": "0.1.0",
+		"@gov.au/body": "0.1.0"
```

Comment:
As you noted @dominikwilkowski we prolly want to specify version ceiling here (and throughout all modules)? (: